### ISSUE TITLE
Agents not appearing

### ISSUE THREAD

**Issue by daven1985** [WaterByWind/grafana-dashboards#13]
I may be doing something really stupid. But I added two of my AP's as below in the telegraf.conf however I can't find where to see the AP's. In Grafana it just lists NONE.

Am I doing something wrong? Cheers.

##
##
 [[inputs.snmp]]
   # List of agents to poll
   agents = [ "172.16.2.128:161", "172.16.2.129:161" ]

**Comment by WaterByWind** [comment]
Have you configured SNMP to be enabled via the UniFi GUI?  SNMP is not enabled by default.

Do you see any errors in the telegraf log?  Typically this might be something like /var/log/telegraf.log.  If you see timeouts or other errors that would be a helpful clue to the next step.

You can run a one-shot test to confirm whether telegraf is able to fetch data:
`/usr/local/bin/telegraf --config /usr/local/etc/telegraf.conf --input-filter snmp --test`

Substitute the proper path to the telegraf binary as well as the telegraf.conf above.  You should see a bit of output displayed.  If not then that's where to check next.

**Comment by daven1985** [comment]
I got it working. In Chronograf I can see the agents I added by their IP address. Do I change them to host names by making sure they Hostname has a DNS reference?

Cheers.

**Comment by WaterByWind** [comment]
Yes you can use hostnames if they are resolvable by the host running telegraf.
